samples/SocialSample/Startup.cs

```diff
@@ -58,6 +61,17 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
             {
                 options.ClientId = "560027070069-37ldt4kfuohhu3m495hk2j4pjp92d382.apps.googleusercontent.com";
                 options.ClientSecret = "n2Q-GEw9RQjzcRbU3qhfTj8f";
+                options.Events = new OAuthEvents()
+                {
+                    OnRemoteError = ctx =>
```

Comment:
I guess that would result in the same 500 most of the time if you don't want to handle those cases. Put it here? https://github.com/aspnet/Security/blob/c14119b612151d0d528b8cdb2bfaa2f11f55ef40/src/Microsoft.AspNet.Authentication/RemoteAuthenticationHandler.cs#L39